src/nls/ko/strings.js

```diff
@@ -494,11 +494,11 @@ define({
     "INVALID_VERSION_NUMBER": "패키지 버전 번호 ({0})가 잘못되었습니다.",
     "INVALID_BRACKETS_VERSION": "{APP_NAME} 호환 문자열 {0}이 잘못되었습니다.",
     "DISALLOWED_WORDS": "{1} {0} 필드에서는 사용할 수 없습니다.",
-    "API_NOT_COMPATIBLE": "확장 기능은이 버전의 {APP_NAME}과 호환되지 않습니다. 잘못된 확장 기능 폴더에 설치됩니다.",
+    "API_NOT_COMPATIBLE": "확장 기능은이 버전의 {APP_NAME}과(와) 호환되지 않습니다. 잘못된 확장 기능 폴더에 설치됩니다.",
     "MISSING_MAIN": "패키지에 main.js 파일이 포함되어 있지 않습니다.",
     "EXTENSION_ALREADY_INSTALLED": "이 패키지를 설치하면 이전에 설치한 확장 기능이 무시됩니다. 이전 확장을 덮어 쓰시겠습니까?",
     "EXTENSION_SAME_VERSION": "이 패키지 버전은 현재 설치되어있는 버전과 동일합니다. 기존 설치를 겹쳐 쓰시겠습니까?",
-    "EXTENSION_OLDER_VERSION": "이 패키지 버전 ({0})는 현재 설치되어있는 버전 ({1})보다 이전 버전입니다. 기존 설치를 겹쳐 쓰시겠습니까?",
+    "EXTENSION_OLDER_VERSION": "이 패키지 버전 ({0})은(는) 현재 설치되어있는 버전 ({1})보다 이전 버전입니다. 기존 설치를 겹쳐 쓰시겠습니까?",
```

Comment:
In this case, the string will be like `"이 패키지 버전(1.2.2)은 현재..."`. That is, the first postposition will match `"버전"`. You should use `"은"` only instead of `"은(는)"`.